src/components/table/Table.styles.ts

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+export const TableStyle = styled.table`
+  width: 90%;
+  border-collapse: collapse;
+  margin: 20px auto 0 auto;
+  background-color: #f0e21e;
+  border-radius: 20px;
+`;
```

Comment:
Acá hice algunas modificaciones para que el diseño quede más similar al de Figma:

```suggestion
export const TableStyle = styled.table`
  width: calc(100% - 160px);
  border-collapse: collapse;
  margin: 20px auto 0 auto;
  background-color: #f0e21e;
  border-radius: 10px;
  table-layout: fixed; 
`;
```